server/src/main/java/org/apache/druid/server/initialization/AuthorizerMapperModule.java

```diff
@@ -58,7 +62,10 @@ public void configure(Binder binder)
     binder.bind(AuthorizerMapper.class)
           .toProvider(new AuthorizerMapperProvider())
           .in(LazySingleton.class);
-
+    binder.bind(new TypeLiteral<Consumer<String>>() {})
```

Comment:
Why not use an interface named `AuthorizerNameValidator`? Then the interface could have javadocs explaining the expected usage and contract.